include/onnxruntime/core/framework/kernel_registry.h

```diff
@@ -37,7 +37,7 @@ class KernelRegistry {
   // TODO(Task:132) Make usage of unique_ptr/shared_ptr as out param consistent
   Status TryCreateKernel(const Node& node, const IExecutionProvider& execution_provider,
                          const std::unordered_map<int, OrtValue>& constant_initialized_tensors,
-                         const OrtValueNameIdxMap& mlvalue_name_idx_map, const FuncManager& funcs_mgr,
+                         const OrtValueNameIdxMap& mlvalue_name_idx_map, FuncManager& funcs_mgr,
```

Comment:
Change funcs_mgr to non-const. Because this function may modify funcs_mgr's internal states.  Previously we hided the fact.